packages/bitcore-node/src/rpc.ts

```diff
@@ -120,4 +120,9 @@ export class RPC {
   async blockAsync(hash: string): Promise<any> {
     return await this.asyncCall('getblock', [hash]);
   }
+
+  getEstimateSmartFee(target: number, callback: CallbackType) {
+    this.callMethod('estimatesmartfee', [target], callback);
```

Comment:
lets switch this part to asyncCall, so we don't have to wrap it in a promise later

Reply:
resolved